mne/tests/test_evoked.py

```diff
@@ -585,7 +585,7 @@ def test_arithmetic():
     evoked2.reorder_channels(evoked2.ch_names[::-1])
     assert not np.allclose(data2, evoked2.data)
     with pytest.warns(RuntimeWarning, match='reordering'):
-        ev3 = grand_average((evoked1, evoked2))
+        ev3 = combine_evoked([evoked1, evoked2], weights=[0.5, 0.5])
```

Comment:
why did you need to change this?

Reply:
because internally, `grand_average` first calls `equalize_channels` and then `combine_evoked`. In current master, `equalize_channels` does not re-order the channels, hence the re-ordering logic in `combine_evoked` is activated, which gives a warning, which is what is tested here. In this PR, `equalize_channels` does re-order the channels, without warning, because I assume re-ordering is something the user wants to do when "equalizing channels", hence the re-ordering logic in `combine_evoked` is not activates, hence no warning is issued, hence the test fails. Since the test is for the re-ordering logic of `combine_evoked`, I thought it better to just call it directly without going through `grand_average`.

Reply:
ok